dbhydra/dbhydra_core.py

```diff
@@ -80,9 +102,14 @@ def new_function(instance, *args, **kw):
     return (new_function)
 
 
+# class DataMigrator:
+
+
+
 class Migrator:
-    def __init__(self, db=None):
+    def __init__(self, db=None, db2=None):
         self.db = db
+        self.db2 = db2
```

Comment:
I don't think this is correct - db2 only needed if migrating data, not migrating structure